### DISCUSSION TITLE
Image update fails with "Changes could not be saved"

### DISCUSSION THREAD

**Discussion by CoreyJ87** [photoprism/photoprism#3686] [Q&A] [answered]
I am trying to change some of my pictures to private and no matter what I get "Changes could not be saved"

<img width="547" alt="Screenshot 2023-08-30 at 3 52 41 AM" src="https://github.com/photoprism/photoprism/assets/247750/acba1c2d-ac7e-4be2-a0fe-61b4c38756ca">
this is what is showing up in the console. 

Edit: Just realized im getting the same errors for any options in the main photoprism settings

**Comment by lastzero** [comment]
Maybe there's a general database error or your server is down?

**Reply by lastzero** [reply]
It could also be a filesystem issue.

**Reply by CoreyJ87** [reply, marked as answer]
That was it. Thanks. I didnt realize the VM ran outa disk space

**Comment by CoreyJ87** [comment]
> It could also be a filesystem issue.
